apps/meteor/client/views/marketplace/components/CategoryFilter/CategoryDropDownAnchor.tsx

```diff
@@ -15,33 +16,38 @@ const CategoryDropDownAnchor = forwardRef<HTMLElement, CategoryDropDownAnchorPro
 
 	return (
 		<Box
-			is={Button}
+			is='button'
 			ref={ref}
 			onClick={onClick}
+			bg={selectedCategoriesCount ? colorTokens.b500 : 'light'}
+			borderColor={selectedCategoriesCount ? 'none' : 'light'}
+			borderRadius='x4'
+			borderWidth={selectedCategoriesCount ? 'none' : 'x1'}
 			display='flex'
-			alignItems='center'
-			justifyContent='space-between'
-			minWidth='x144'
 			flexGrow={1}
 			flexShrink={1}
-			borderColor={selectedCategoriesCount ? 'none' : 'light'}
-			borderWidth={selectedCategoriesCount ? 'none' : 'x1'}
-			{...(selectedCategoriesCount ? { primary: true } : { bg: 'surface-light' })}
+			justifyContent='space-between'
+			minWidth='x144'
+			pb={9}
+			pie={7}
+			pis={14}
 			{...props}
 		>
 			{selectedCategoriesCount > 0 && (
 				<Box
-					mie={6}
-					borderRadius='x32'
+					is={Button}
```

Comment:
why is this a `Button`? It's not semantically correct to have a **button inside a button**

We do have a similar component: `Badge` that I would suggest applying here. The design is not exactly the same, but we should avoid having **slightly** different and customized components all around...